app/models/audit/bars/BarsVerifyDetails.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.audit.bars
+
+import play.api.libs.json.{Json, OWrites}
+
+final case class BarsVerifyDetails(attempts: Int, lockoutExpiryDateTime: Option[String])
```

Comment:
for consistency?
```scala
attempts: NumberOfBarsVerifyAttempts,
```

Reply:
I've tried to keep everything to the most basic types in the audit events so it's clearer how things will get written into the JSON - with custom types there's sometimes custom serialisers which aren't what has been agreed with the CIP team. It wouldn't be the case here though cos that type would write like a simple JsNumber

Reply:
I was just about to delete my comment